### DISCUSSION TITLE
Query ProjectV2SingleSelectField with GraphQL not working

### DISCUSSION THREAD

**Discussion by a7brusco** [community/community#50253] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Context
I've been experiencing what could be done with the GitHub GraphQL API and webhooks (especially for the new projects, called projects V2).
I get the body of the webhook sent by a private repo as a JSON file, and I run custom automation using some python scripts.
Here is an example (actually of part of it) of the webhook body i'm using:
```JSON
{
    "action": "edited",
    "projects_v2_item": {
        "content_node_id": "blablabla",
        "node_id": "blablabla",
        "project_node_id": "blablabla"
    },
    "changes": {
        "field_value": {
            "field_node_id": "blablabla"
        }
    }
}
```
The `field_node_id` refers to the string ID of the field edited in a project V2.
In my python script, I need to know the value of that field.
To do that, I use 2 GraphQL requests.
The first one query the name of the field edited and the second one query the value thanks to its name.
I couldn't find a way to get the field value without the field name.
Here is the first GraphQL request, to query the field name:
```graphql
query {
    node(id: "<field_id_from_webhook>") {
        ... on ProjectV2SingleSelectField {
            id
            name
        }
    }
}
```

### The issue
Here is the response of to the query:
```JSON
{
  "data": {
    "node": null
  },
  "errors": [
    {
      "type": "NOT_FOUND",
      "path": [
        "node"
      ],
      "locations": [
        {
          "line": 3,
          "column": 5
        }
      ],
      "message": "Could not resolve to a node with the global id of '<field_id_from_webhook>'"
    }
  ]
}
```

However, when I search "manually" for the field name, I can find it:

```graphql
query {
    organization(login: "<myorg>") {
        projectV2(number: 4) {
            field(name: "Status") {
                ... on ProjectV2SingleSelectField {
                    id
                    name
                    dataType
                }
            }
        }
    }
}
```
output:
```JSON
{
  "data": {
    "organization": {
      "projectV2": {
        "field": {
          "id": "<field_id_from_webhook>",
          "name": "Status",
          "dataType": "SINGLE_SELECT"
        }
      }
    }
  }
}
```
### The questions

- Am I missing a piece of information about field IDs or else in the GraphQL API ?
- Has anyone also experienced this curious behavior ?
- How can I get rid of this issue and use another way to get the field name by its ID ?

### What I'm sure of

It's not a permission issue, I've already tested it using the [GitHub Explorer](https://docs.github.com/en/graphql/overview/explorer) and other HTTP client.

**Comment by uberdeveloper** [comment]
Quite late to this post
You are actually querying the `projectV2Item`
try this query
```graphql
{
  node(id: "node_id") {
    ... on ProjectV2Item {
      id
      databaseId
      content {
        ... on DraftIssue {
          title
          body
        }
      }
    }
  }
}
```
Here I am fetching the item and its content, the content can also be queried directly
